apps/frontend/src/components/dashboard/DashboardHeader.tsx

```
@@ -33,110 +44,282 @@ export function DashboardHeader({
+                  size="sm"
+                  onClick={onToggleAnalytics}
+                  className="gap-2"
+                  aria-pressed={showAnalytics}
+                >
+                  <TrendingUp className="w-4 h-4" />
+                  <span className="hidden sm:inline">Analytics</span>
+                </Button>
+              )}
+            </div>
+            <p className="hidden sm:block text-sm text-muted-foreground">
+              Welcome back! You're logged in as {roleLabels[userRole]}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)